src/pages/ResetPage.js

```diff
@@ -61,45 +61,67 @@ const ResetPage = () => {
+                    value: /(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{6,}/,
+                    message: 'A password deve ter pelo menos 6 caracteres e conter pelo menos um número, uma letra minúscula e uma letra maiúscula.',
+                  },
+                }}
+                render={({ field }) => (
+                  <TextField
+                    label='Nova password'
+                    type={componentProps.type.password}
+                    variant={componentProps.variant.outlined}
+                    fullWidth
+                    sx={resetClasses.field}
+                    error={errors.password ? true : false}
+                    disabled={isLoading}
+                    autoComplete='true'
+                    {...field}
+                  />
+                )}
               />
-
-              <TextField
-                label='Repetir Password'
-                type='password'
-                variant={componentProps.variant.outlined}
-                fullWidth
-                required
-                sx={resetClasses.field}
-                onChange={(e) => setNewPassword2(e.target.value)}
-                error={passwordError}
+              <Controller
+                name={componentProps.name.password2}
+                control={control}
+                rules={{ required: 'Password em falta', validate: (value) => value === password || 'Insira a mesma password nos 2 campos' }}
+                render={({ field }) => (
+                  <TextField
+                    label='Repetir Password'
+                    type={componentProps.type.password}
+                    variant={componentProps.variant.outlined}
+                    fullWidth
+                    sx={resetClasses.field}
+                    error={errors.password2 ? true : false}
+                    disabled={isLoading}
+                    autoComplete='true'
+                    {...field}
+                  />
+                )}
               />
 
               {errorMessage && <ErrorMessage message={errorMessage} />}
+              {errors.password && <ErrorMessage message={errors.password.message} />}
+              {errors.password2 && <ErrorMessage message={errors.password2.message} />}
 
-              <Button variant={componentProps.variant.contained} type={componentProps.type.submit}>
-                Repor
-              </Button>
+              {!isLoading && (
+                <Button variant={componentProps.variant.contained} type={componentProps.type.submit}>
+                  Repor
+                </Button>
+              )}
             </form>
           </Box>
-        </>
+        </Box>
       )}
       {successMessage && (
         <Box sx={resetClasses.top}>
-          <Box sx={resetClasses.image}>
-            <img src={resetImage} alt='Forgot password' className='auth-images' />
-          </Box>
           <SuccessMessage message={successMessage} />
+          <Button onClick={() => navigate('/login')}>Entrar</Button>
```

Comment:
I would create a global file to store the route names and then just call `Routes.Login`